### ISSUE TITLE
Excepted as: Error when checking input: expected input_1 to have shape (80, 320, 3) but got array with shape (160, 320, 3)

### ISSUE THREAD

**Issue by ghost** [cpgeier/DeepGTAV_Tools#8]
Hey I get this error:
Excepted as: Error when checking input: expected input_1 to have shape (80, 320, 3) but got array with shape (160, 320, 3) do you know why ?

**Comment by ghost** [comment]
I changed 
 image = frame2numpy(message['frame'], (320,160))
to
 image = frame2numpy(message['frame'], (320,80))
and it worked 
but I get only -0.998 ad Category 1 Decimals
Any idea why ?

**Comment by nanxiangshui** [comment]
frame2numpy() only decodes images from data you have recoreded. Simply changing the size of decoding is not appropriate.

According to my experiments, I think a proper way to deal with this issue is to remove the upper half of the input image, i.e.,
image = image[80:, :, :]

Hope that help

**Comment by hemanth2410** [comment]
yeah, I am experiencing the same issue, Deep GTA is actually sending only upper half of screen.
so you are getting error "Error when checking input: expected input_1 to have shape (80, 320, 3) but got array with shape (160, 320, 3) "

**Comment by liushaowei123** [comment]
author, can you show how you choice the image from 160 to 80
